packages/pyo3/pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
```

Comment:
This is the file that tells sdist how to build our system.  I'm still not confident it works exactly, but we'll see!